module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -458,45 +471,56 @@ public function getHolding($bibId, array $patron = null, array $options = [])
      */
     public function patronLogin($username, $password)
     {
-        // Get user id
-        $query = ['query' => 'username == ' . $username];
-        $response = $this->makeRequest('GET', '/users', $query);
+        if ($this->config['User']['okapi_login'] ?? false) {
+            $tenant = $this->config['API']['tenant'];
+            $credentials = compact('tenant', 'username', 'password');
```

Comment:
The `compact` function is a handy shorthand way of bundling a bunch of variables into an array. In this instance, `compact('tenant', 'username', 'password')` is the same as saying:

<pre>
[
    'tenant' => $tenant,
    'username' => $username,
    'password' => $password,
];
</pre>

I used it here to save space, and I also reintroduced it below to make the query construction a little simpler.